src/runtime/types.ts

```diff
@@ -395,6 +395,11 @@ export interface SecondarySignInOptions extends Record<string, unknown> {
    * @default false
    */
   external?: boolean;
+  /** Prevent the signIn flow during registration and return the signUp response
+   *
+   * @default false
+   */
+  stopLogin?: boolean;
```

Comment:
This is something we only want to use for the registration flow, but adding it here, would also allow you to pass it for the signin flow. I think we need to find a way to:

- Not modify the signin options (as for registration these are then passed onto the `signIn` function
- We don't want to change the order of props that can be passed (to not make breaking changes)

I think we need to either: 
- Expand the `signInOptions` type by creating a `registerOptions` type with the added value
- pass the value as a third optional parameter

Also lets rename it to `preventLoginFlow`